### PR TITLE
Revert "fix: Remove duplicate word as a contributor"

### PR DESCRIPTION
**Pull request by CBID2**
Reverts open-sauced/guestbook#164

### CODE REVIEW

File: .github/workflows/intro_course_contributor.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: Intro Course Contributor Action
 
 on:
+ on:
```
Comment (adiati98) [inline .github/workflows/intro_course_contributor.yml:4]: This is a duplication.

**Review by adiati98 — COMMENTED** [review]
@CBID2 I think @BekahHW submitted the last PR to remove this duplication that caused the GitHub Action run failed 🙂

File: .github/workflows/intro_course_contributor.yml (hunk 1 above)
Comment (CBID2) [reply .github/workflows/intro_course_contributor.yml:4]: Ohh

Comment (CBID2) [reply .github/workflows/intro_course_contributor.yml]: Did it @adiati98

**Comment by BekahHW** [comment]
Can we close this since nothing has changed?

**Pull request closed without merging**